From a0739ed017cd097817d227e98848f66c8e5d666e Mon Sep 17 00:00:00 2001
From: Eric Nelson <eric.nelson@boundarydevices.com>
Date: Tue, 11 Dec 2012 11:59:47 -0700
Subject: [PATCH] nitrogen6x bootscript: modify for use with the Yocto Project

Change 6x_bootscript to match the partitioning scheme currently in
use by meta-fsl-arm:

	- Set root filesystem to partition 2 (/dev/mmcblk0p2)
	- Load kernel from root of partition 1

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Eric Nelson <eric.nelson@boundarydevices.com>
---
 board/boundary/nitrogen6x/6x_bootscript.txt |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/board/boundary/nitrogen6x/6x_bootscript.txt b/board/boundary/nitrogen6x/6x_bootscript.txt
index fe75976..1d766ca 100644
--- a/board/boundary/nitrogen6x/6x_bootscript.txt
+++ b/board/boundary/nitrogen6x/6x_bootscript.txt
@@ -53,6 +53,6 @@ done
 
 setenv bootargs $bootargs $fbmem
 
-setenv bootargs $bootargs console=ttymxc1,115200 vmalloc=400M root=/dev/mmcblk0p1 rootwait consoleblank=0 ;
-${fs}load mmc ${disk}:1 10800000 /boot/uImage && bootm 10800000 ; echo "Error loading kernel image"
+setenv bootargs $bootargs console=ttymxc1,115200 vmalloc=400M root=/dev/mmcblk0p2 rootwait consoleblank=0 ;
+${fs}load mmc ${disk}:1 10800000 uImage && bootm 10800000 ; echo "Error loading kernel image"
 
-- 
1.7.10.4

